website/content/docs/intro/index.mdx

```diff
@@ -49,71 +41,50 @@ Review the video below to learn more about Consul from HashiCorp's co-founder Ar
-where data is stored and replicated. The servers themselves elect a leader.
-While Consul can function with one server, 3 to 5 is recommended to avoid
-failure scenarios leading to data loss. A cluster of Consul servers is recommended
-for each datacenter.
-
-The servers maintain a _catalog_, which is formed by aggregating information
-submitted by the agents. The catalog maintains the high-level view of the cluster,
-including which services are available, which nodes run those services, health
-information, and more. How agents and the catalog interact can be found
-[here](/docs/architecture/anti-entropy#catalog).
-
-Components of your infrastructure that need to discover other services
-or nodes can query any of the Consul servers _or_ any of the Consul agents.
-The agents forward queries to the servers automatically.
-
-Each datacenter runs a cluster of Consul servers. When a cross-datacenter
-service discovery or configuration request is made, the local Consul servers
-forward the request to the remote datacenter and return the result.
-
-## Next Steps
-
-Continue onwards with [HashiCorp Learn](https://learn.hashicorp.com/tutorials/consul/get-started-install)
-to learn more about Consul and how to get Consul up and running.
+### Automate service discovery
+
+Adopting a microservices architecture on cloud infrastructure is a critical step toward delivering value at scale, but knowing where healthy services are running on your networks in real time becomes a challenge. Consul automates service discovery by replacing service connections usually handled with load balancers with an identity-based service catalog. The service catalog is a centralized source of truth that you can query through Consul’s DNS server or API. The catalog always knows which services are available, which have been removed, and which services are healthy.
+
+### Connect services across runtimes and cloud providers
+
+The shift to cloud has evolved. Organizations have adopted a multi-cloud strategy where customer value is delivered on various runtimes and cloud providers, which can lead to inconsistency and technical debt. Consul routes network traffic to any runtime or infrastructure environment your services need to reach using proxies. You can also use Consul API Gateway to route traffic into and out of the network. Consul service mesh provides additional capabilities, such as securing communication between services, traffic management, and observability, with no application code changes.
+
+Consul also has many integrations with Kubernetes that enable you to leverage Consul features in containerized environments. For example, Consul can automatically inject sidecar proxies into Kubernetes Pods and sync Kubernetes Services and non-Kubernetes services into the Consul service registry. 
+
+You can also schedule Consul workloads with [HashiCorp Nomad](https://www.nomadproject.io/) to provide secure service-to-service communication between Nomad jobs and task groups. 
+
```

Comment:
"And because all of these environments and runtimes are first-class participants within Consul, a single Consul deployment can connect together any number of them, instead of having isolated, environment- or cloud-specific solutions, and without forcing workloads in one runtime to have to pretend to be in a completely different runtime."

Additional, but I think very key, point here